.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I can't spot a difference to the previous config. What's the idea behind this change?

Reply:
> Moved .eslintrc.js to .eslintrc.json because some the older ESLint versions this library depends would fail to require it now that the closest package.json is of "type": "module".